### PR TITLE
Suit sensors can now be quick-maxed by ctrl clicking

### PR DESCRIPTION
**Pull request by grungussuss**
## About The Pull Request
Ctrl clicking your jumpsuit will max its suit sensors

https://github.com/tgstation/tgstation/assets/96586172/3b7275ee-404a-49d1-b378-63de794d22ee

## Why It's Good For The Game
Nice little change that makes maxing your sensors easier

## Changelog

:cl: grungussuss
qol: suit sensors can now be maxed by ctrl clicking your jumpsuit
/:cl:

### CODE REVIEW

File: code/modules/clothing/under/_under.dm [hunk 1]
```diff
@@ -337,6 +338,14 @@
 		if(H.w_uniform == src)
 			H.update_suit_sensors()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	. = ..()
+	if(!can_toggle_sensors(user))
+		return
+
+	src.sensor_mode = SENSOR_COORDS
+	to_chat(user, span_notice("Your suit will now report your exact vital lifesigns as well as your coordinate position."))
```
Comment (Mothblocks) [inline code/modules/clothing/under/_under.dm:347]: balloon

**Review by Mothblocks — CHANGES_REQUESTED** [review]
imo it should toggle between maxxed/not maxxed

File: code/modules/clothing/under/_under.dm [hunk 2]
```diff
@@ -337,6 +338,14 @@
 		if(H.w_uniform == src)
 			H.update_suit_sensors()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	. = ..()
+	if(!can_toggle_sensors(user))
```
[the remark this answers is not in the document]
Comment (Sadboysuss) [reply code/modules/clothing/under/_under.dm:343]: image
image
should be working fine
did you test if you can do that while in hard crit?

**Comment by Sadboysuss** [comment]
> imo it should toggle between maxxed/not maxxed

I don't know how to do that so I'd like to leave it out of scope

File: code/modules/clothing/under/_under.dm (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sadboysuss) [reply code/modules/clothing/under/_under.dm:343]: it doesn't work while in crit, tested.

https://github.com/tgstation/tgstation/assets/96586172/908ecbc8-af19-4b1b-afaf-7edfa26fc19b

[the remark this answers is not in the document]
Comment (Iamgoofball) [reply code/modules/clothing/under/_under.dm:343]: huh, idk then

File: code/modules/clothing/under/_under.dm [hunk 3]
```diff
@@ -344,7 +344,7 @@
 		return
 
 	src.sensor_mode = SENSOR_COORDS
-	balloon_alert(user, "Your suit will now report your exact vital lifesigns as well as your coordinate position.")
+	balloon_alert(user, "your suit will now report your exact vital lifesigns as well as your coordinate position.")
```
Comment (Iamgoofball) [inline code/modules/clothing/under/_under.dm:347]: way waaaaay too long, change it to "coordinate sensors"

File: code/modules/clothing/under/_under.dm [hunk 4]
```diff
@@ -69,6 +69,7 @@
 
 	if(isnull(held_item) && has_sensor == HAS_SENSORS)
 		context[SCREENTIP_CONTEXT_RMB] = "Toggle suit sensors"
+		context[SCREENTIP_CONTEXT_CTRL_LMB] = "Set suit sensors to coordinates"
```
Comment (ZephyrTFA) [inline code/modules/clothing/under/_under.dm:72]: People don't know what "coordinates" means.
They are being set to `Tracking`.

File: code/modules/clothing/under/_under.dm [hunk 5]
```diff
@@ -337,6 +338,14 @@
 		if(H.w_uniform == src)
 			H.update_suit_sensors()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	. = ..()
+	if(!can_toggle_sensors(user))
+		return
+
+	src.sensor_mode = SENSOR_COORDS
+	balloon_alert(user, "your suit will now report your exact vital lifesigns as well as your coordinate position.")
```
Comment (ZephyrTFA) [inline code/modules/clothing/under/_under.dm]: A balloon alert:
- should not be longer than five words 
- should not end in punctuation except for `...` and `!` and neither of these fit this scenario

File: code/modules/clothing/under/_under.dm [hunk 6]
```diff
@@ -337,6 +338,14 @@
 		if(H.w_uniform == src)
 			H.update_suit_sensors()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	. = ..()
+	if(!can_toggle_sensors(user))
+		return
+
+	src.sensor_mode = SENSOR_COORDS
```
Comment (ZephyrTFA) [inline code/modules/clothing/under/_under.dm:346]: you do not need `src.` here

File: code/modules/clothing/under/_under.dm [hunk 7]
```diff
@@ -337,6 +338,14 @@
 		if(H.w_uniform == src)
 			H.update_suit_sensors()
 
+/obj/item/clothing/under/CtrlClick(mob/user)
+	. = ..()
```
Comment (ZephyrTFA) [inline code/modules/clothing/under/_under.dm:342]: check for parent returning `FALSE` and return.

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**